VTUL/iawa
# LIBTD-1821: Display top level /user collections with representative thumbnails

Remove DOI in bibliographic_citation

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1821) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)
* https://webapps.es.vt.edu/jira/browse/LIBTD-1823

# What does this Pull Request do? (:star:)
Filter out top level user collection when exploring collections; also remove DOI info. in bibliographic_citation metadata

# What's the changes? (:star:)

* Override collection_thumbnail_path_service to set the thumbnail
* Only get collections when nesting level is 1
* Remove the "DOI" text from bibliographic_citation

# How should this be tested?

* Run as railsapps: bundle exec rake iawa:batch_import["Ms1990_025_Rudoff.index.csv"]
* Click on "Explore Collection" from the Menu in home page, should only display top level collections with thumbnail image properly displayed
* Login as Admin, go to dashboard, edit a collection, the bibliographic_citation auto-filled field should have no DOI information

# Interested parties
Tag (@pmather ) interested parties

(:star:) Required fields
